src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1516,6 +1516,8 @@ internal enum ErrorCode
         WRN_NullabilityMismatchInParameterTypeOfTargetDelegate = 8622,
         ERR_ExplicitNullableAttribute = 8623,
         ERR_NotNullableOperatorNotReferenceType = 8624,
+        WRN_NoBestNullabilityArrayElements = 8625,
+        WRN_NoBestNullabilityConditionalExpression = 8626,
```

Comment:
>WRN_NoBestNullabilityConditionalExpression [](start = 8, length = 42)

Consider moving the entire block of Nullable errors to the end so that the sort order for values is preserved.